### PR TITLE
feat: add vault handler manifest files

### PR DESCRIPTION
**Pull request by muse-sisay**
## Description
This PR moves https://github.com/kubefirst/manifests/tree/main/vault-handler to this repo.

## Related Issue(s)

Fixes #

### CODE REVIEW

File: vault-handler/replicas-1/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+namespace: vault
+
+resources:
+  - job.yaml
+  - rbac.yaml
+
+images:
+  - name: kubefirst/vault-handler
+    newName: ghcr.io/konstructio/vault-handler
+    newTag: 'v1.0.0'
```
Comment (patrickdappollonio) [inline vault-handler/replicas-1/kustomization.yaml]: I have to wonder if we can remove that and instead point to the actual image overall

File: vault-handler/replicas-3/job.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  name: vault-handler
+  namespace: vault
+spec:
+  template:
+    spec:
+      serviceAccountName: vault-handler
+      containers:
+        - name: vault-handler
+          image: kubefirst/vault-handler
```
Comment (patrickdappollonio) [inline vault-handler/replicas-3/job.yaml]: Changing the image here...

**Review by patrickdappollonio — DISMISSED** [review]

File: vault-handler/replicas-1/kustomization.yaml (hunk 1 above)
Comment (muse-sisay) [reply vault-handler/replicas-1/kustomization.yaml]: Done Señor!

File: vault-handler/replicas-3/job.yaml (hunk 2 above)
Comment (muse-sisay) [reply vault-handler/replicas-3/job.yaml]: Fixed, thank you for pointing it out.

**Review by patrickdappollonio — APPROVED** [review]

**Pull request merged**